api/actions/__test__/createWorkItem.spec.js

```diff
@@ -348,7 +471,7 @@ describe('createGusItem action', () => {
         expect(Github.createComment).toHaveBeenCalledWith({
             req,
             body:
-                'Sorry we could wait until Heroku connect make the syncronization.'
+                'Sorry we could wait until Heroku connect make the synchronization.'
```

Comment:
Fixed a typo.

Reply:
is this message saying we could *not* wait until heroku connect could make the syncronization? or that we had to wait until the heroku connect could make the synchronization?

Reply:
Good point! I should be "could **not**". @reiniergs - thoughts ?